src/views/Clients/Clients.vue

```diff
@@ -323,6 +334,50 @@ const handlePageChange = (no: number) => {
   loadNodeClients(isBack)
 }
 
+const handleExactSearchClient = async (params: Record<string, any>) => {
+  try {
+    const {
+      clientid,
+
+      username,
+      ip_address,
+      conn_state,
+      like_username,
+      gte_connected_at,
+      lte_connected_at,
+    } = params
+    const data = await exactSearchClient(clientid)
+    let isMatchOther = true
+    if (username && data.username !== username) {
+      isMatchOther = false
+    }
+    if (ip_address && data.ip_address !== ip_address) {
+      isMatchOther = false
+    }
+    if (conn_state && data.connected !== (conn_state === 'connected')) {
+      isMatchOther = false
+    }
+    if (like_username && data.username.indexOf(like_username) === -1) {
+      isMatchOther = false
+    }
+    if (
+      gte_connected_at &&
+      new Date(data.connected_at).getTime() < new Date(gte_connected_at).getTime()
+    ) {
+      isMatchOther = false
+    }
+    if (
+      lte_connected_at &&
+      new Date(data.connected_at).getTime() > new Date(lte_connected_at).getTime()
+    ) {
+      isMatchOther = false
+    }
+    return Promise.resolve({ data: isMatchOther ? [data] : [], meta: { count: 1 } })
+  } catch (error) {
+    return Promise.reject(error)
+  }
+}
```

Comment:
so many `if` statements. I think maybe we can make some improvements to this function?

for the example:

```js

const handleExactSearchClient = async (params: Record<string, any>) => {
  const {
    clientid,
    username,
    ip_address,
    conn_state,
    like_username,
    gte_connected_at,
    lte_connected_at,
  } = params;

  try {
    const data = await exactSearchClient(clientid);
    
    // 检查所有条件
    const isMatch = 
      (!username || data.username === username) &&
      (!ip_address || data.ip_address === ip_address) &&
      (!conn_state || data.connected === (conn_state === 'connected')) &&
      (!like_username || (data.username && data.username.indexOf(like_username) !== -1)) &&
      (!gte_connected_at || new Date(data.connected_at).getTime() >= new Date(gte_connected_at).getTime()) &&
      (!lte_connected_at || new Date(data.connected_at).getTime() <= new Date(lte_connected_at).getTime());

    return {
      data: isMatch ? [data] : [],
      meta: { count: 1 }
    };
  } catch (error) {
    return Promise.reject(error);
  }
};
```